### ISSUE TITLE
Sentry receiving two API Calls from Logger

### ISSUE THREAD

**Issue by rainerCH** [notamedia/yii2-sentry#41]
Hi,

i am using your library (thanks!) in yii2 and my sentry is receiving always two POST requests (and Events in Sentry) from my application.

10.42.2.0 - - [09/Jul/2020:12:22:00 +0000] "POST /api/2/store/ HTTP/1.1" 200 366 "-" "sentry.php/2.3.2"
10.42.2.0 - - [09/Jul/2020:12:22:00 +0000] "POST /api/2/store/ HTTP/1.1" 200 366 "-" "sentry.php/2.3.2"

my config is very basic:

```php
    $config['components']['log']['targets'][] = [
        'class' => 'notamedia\sentry\SentryTarget',
        'dsn' => 'https://***@***/2',
        'levels' => ['error', 'warning'],
        'except' => ['yii\web\HttpException:401'],
        'context' => true,
        'extraCallback' => function($message, $extra) {
            $extra['request_id'] = \Yii::$app->requestId->get();
            return $extra;
        }
    ];
```

one event has all the extra infos in it like user, ip... and the other one is very basic. i am confused :)

could you reproduce this problem? 

thanks!
Rainer

**Comment by CookiesEater** [comment]
Hi! I checked with provided config but could not reproduce (always it was 1 event in sentry)

Can you check whole Yii config, please? May be there are have 2 target for sentry

**Comment by aebnoether-backoffice** [comment]
(Me and @rainerCH are work colleagues and we work on the same problem)

I further investigated this Issue. In my opinion it is more than likely that the Issue is related to this change: https://github.com/notamedia/yii2-sentry/commit/3d2676786fd9bbfcca88dc98044048bab924c26a#diff-6c9daa1dd1ea6db50b80e139ed98b740R46

For some reason Sentry is initiated as soon as the constructor of `notamedia\sentry\SentryTarget` is called. When Sentry is initiated it registers an exception handler which is called upon the occurrence of any PHP Exception. When a PHP-Exception occurrs, first the exception handler of Sentry is called and it pushes the exception to the sentry server. Then the Yii2 Logging mechanism triggers which in turn uses this package as a target. That is why all exceptions are pushed two times to Sentry.

Before we used Version 1.5.0 and this problem did not occurr. This fact further increases my confidence that the problem is related to the said change.

Why is Sentry initiated when the constructor of `notamedia\sentry\SentryTarget` is called? Is there a way to disable this behaviour?

Thanks in advance!

**Comment by CookiesEater** [comment]
Thank you for paying attention to this!

Init Sentry in construct is needed to configure Sentry Scope before error was handled (for example, add to all error some additional data in global config)

This problem was caused because Sentry has it's own integration (https://docs.sentry.io/platforms/php/default-integrations/) to catch PHP errors

Can you check please this issue with fix in https://github.com/notamedia/yii2-sentry/pull/42 ?

**Comment by rainerCH** [comment]
Hi @CookiesEater 

seems to work for me :) 

thanks!
Rainer

**Comment by CookiesEater** [comment]
Fixed in 1.5.1. Thanks for participating!
